generic/nstress.py

```diff
@@ -60,7 +60,7 @@ def test(self):
         if process.system("/ncpu -p %s -s %s" %
                           (self.procs, self.duration),
                           ignore_status=True, sudo=True) != 1:
-            self.log.info("ncpu test failed")
+            self.fail("ncpu test failed")
```

Comment:
@PraveenPenguin this should be in the other commit.
Also, these all changes can be in a single commit.

Reply:
@narasimhan-v pushed the changes